src/redux/reducers.js

```diff
@@ -5,6 +5,7 @@ const {request, response, fail} = ActionHelpers
 
 import _ from 'lodash'
 import {userParser, postParser} from '../utils'
+import {frontendPreferences as frontendPrefsConfig} from '../config'
```

Comment:
Thanks, I didn't know there's a problem. I guess we'll need to refactor this thing everywhere when migrating to Webpack 2.